src/osd/osdnet.cpp

```diff
@@ -55,7 +55,13 @@ void osd_netdev::stop()
 
 int osd_netdev::send(uint8_t *buf, int len)
 {
-	return 0;
+	/*
+	 * Prevent a time-travel issue by delaying the next receive by one packet,
+	 * giving enough time for the current transmit to complete.
+	 */
+	m_timer->reset(m_timer->elapsed() + m_timer->remaining());
```

Comment:
I think this is the wrong solution.  You should be delaying delivering the transmitted frame to tap/tun by the amount of time it would take for the emulated MAC to get it on the wire.  That depends on the PHY and also how the device works.  Instead you're assuming the emulated MAC can always deliver the entire frame at the point where transmission starts, and kind of enforcing half duplex behaviour as well,